### PR TITLE
refact/code

### PR DESCRIPTION
**Pull request by takayukioda**
some updates :p
FYI @TsubasaK111

### CODE REVIEW

**Comment by takayukioda** [comment]
I haven't update any since last commit, so I'll ask review for now.
@TsubasaK111 review me

File: build.py [hunk 1]
```diff
@@ -152,6 +136,10 @@ def isExceptionTopIndex(self, attr):
 
 # ============================================================================
 
+cwd = str(os.getcwd())
+lsResults = os.listdir(cwd)
+
+isMkdocsInstalled = subprocess.check_output(['mkdocs', '--version']).startswith(b'mkdocs')
```
Comment (TsubasaK111) [inline build.py:142]: These declarations originally were right here,   
but I moved these variables up to the top in `#19f5183` to improve readability.
(There were too many important variables that were declared inside of this code block.)

Considering how we will probably divide `PostMkdocsParser` into a separate file, I think this is a moot point; they will be located in about the same area, at the top where variables should be declared. LGTM.

**Review by TsubasaK111 — APPROVED** [review]
LGTM, and thank you!
On a separate note, this file is becoming too long.
We should divide PostMkdocsParser into a separate file.

**Comment by TsubasaK111** [comment]
LGTM

**Pull request merged**